### PR TITLE
H1 text cutting off on schools fundraising ideas pages on iPads Pro portrait view

### PR DESCRIPTION
**Pull request by LouiaR**
screenshot-live browserstack com-2019 12 23-13_44_55

Issue is here: https://github.com/comicrelief/comicrelief-contentful/issues/448

### CODE REVIEW

File: src/theme/shared/global.css [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
 html {
+  -webkit-box-sizing: border-box;
   box-sizing: border-box;
 }
+
 *,
 *:before,
 *:after {
+  -webkit-box-sizing: inherit;
```
Comment (gusliedke) [inline src/theme/shared/global.css]: do we need to add vender prefix?

Comment (LouiaR) [reply src/theme/shared/global.css]: No really, it added automatically by the editor

Comment (LouiaR) [reply src/theme/shared/global.css]: Removed

**Pull request merged**